eZ/Publish/Core/settings/storage_engines/cache.yml

```diff
@@ -56,6 +63,13 @@ services:
             - "%ezpublish.spi.persistence.cache.inmemory.limit%"
             - "%ezpublish.spi.persistence.cache.inmemory.enable%"
 
+    ezpublish.spi.persistence.cache.inmemory.content:
+        public: false
+        class: eZ\Publish\Core\Persistence\Cache\InMemory\InMemoryCache
+        arguments:
+            - "@=container.getParameter('kernel.debug') ? parameter('ezpublish.spi.persistence.cache.inmemory.content.ttl')  *2 : parameter('ezpublish.spi.persistence.cache.inmemory.content.ttl')"
```

Comment:
1. doesn't this work with just a `%kernel.debug%`?
2. `ezpublish.spi.persistence.cache.inmemory.content.ttl` should be overridden by dedicated config for debugging, not like this (then you could avoid this whole expression).

Reply:
Unsure if I get 1.

On 2, wouldn't I still need the expression language usage? Only thing I can save is the `* 2` part of it.

Reply:
In general I meant [something like this](https://github.com/ezsystems/ezpublish-kernel/compare/inmemory-content-meta...alongosz:inmemory-content-meta-no-computing).

Besides readability, I don't like the fact that this is computed on the fly w/o possibility to override.

(but if you feel we need to impose `*2` always, you can disregard along with point 1., which doesn't work) :)

Reply:
So, then we rather drop this expression and consider as followup to give example with higher ttl in confg_dev.yml then. less magic going on, including compiler pass magic ;)

Reply:
> So, then we rather drop this expression and consider as followup to give example with higher ttl in confg_dev.yml then. less magic going on, including compiler pass "magic" ;)

That also works for me :)